app/uk/gov/hmrc/valuationofficeagencycontactfrontend/filters/SessionIdFilter.scala

```diff
@@ -44,38 +45,23 @@ class SessionIdFilter (
 
     if (rh.session.get(SessionKeys.sessionId).isEmpty) {
 
-      val cookies: String = {
-
-        val session: Session =
-          rh.session + (SessionKeys.sessionId -> sessionId)
-
-        val cookies =
-          rh.cookies ++ Seq(Session.encodeAsCookie(session))
-
-        Cookies.encodeCookieHeader(cookies.toSeq)
-      }
+      val session: Session =
+        rh.session + (SessionKeys.sessionId -> sessionId)
```

Comment:
Aren't the cookies needed anymore?

Reply:
We have same solution on RALD. I didn't want to go to much deep.